CODEOWNERS

```diff
@@ -705,6 +705,7 @@ build.json @home-assistant/supervisor
 /homeassistant/components/irish_rail_transport/ @ttroy50
 /homeassistant/components/isal/ @bdraco
 /tests/components/isal/ @bdraco
+/homeassistant/components/iskra/ @iskramis
```

Comment:
Please run hassfest again, it should also add the tests as codeowned

Reply:
done